docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -113,37 +126,79 @@ You can also customize the content of the header, using the following parameters
 @Composable
 fun MessageListHeader(
     leadingContent: @Composable RowScope.() -> Unit = {
-        BackButton(
-            modifier = Modifier
-                .size(36.dp),
-            imageVector = Icons.Default.ArrowBack,
-            onBackPressed = onBackPressed,
-        )
+        DefaultMessageListHeaderLeadingContent(onBackPressed = onBackPressed)
     },
-
     centerContent: @Composable RowScope.() -> Unit = {
         DefaultMessageListHeaderCenterContent(
-            modifier = Modifier.weight(6f),
+            modifier = Modifier.weight(1f),
             channel = channel,
+            currentUser = currentUser,
+            typingUsers = typingUsers,
             messageMode = messageMode,
             onHeaderActionClick = onHeaderActionClick,
             connectionState = connectionState
         )
     },
     trailingContent: @Composable RowScope.() -> Unit = {
+        DefaultMessageListHeaderTrailingContent(
+            channel = channel,
+            currentUser = currentUser
+        )
+    },
+)
+```
+
+* `leadingContent`: Represents the leading content part of the header. By default shows a back button.
+    * override the default behavior but keep the UI by overriding `onBackPressed`
```

Comment:
I would start these items with a Capital letter and add a full stop at the end.

We also need to rephrase the end of each of the outer items.

E.g. I'd put it as:

```
Represents the leading content part of the header. By default shows a back button and allows you to:
....
```

Instead of just starting a bullet list without any comma or transition

Reply:
Decided on a different heading altogether in a Slack discussion.